internal/characters/sayu/burst.go

```diff
@@ -74,21 +74,23 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 		// first tick is at 145
 		for i := 145 - tickTaskDelay; i < 145+540-tickTaskDelay; i += 90 {
 			c.Core.Tasks.Add(func() {
-				// look for char with lowest HP to heal
-				char := c.Core.Player.LowestHPChar()
-				hasPlayer := c.Core.Combat.Player().IsWithinArea(burstArea) && char.HPCurrent/char.MaxHP() <= 0.7
-
+				// check for player
+				// only check HP of active character
+				char := c.Core.Player.ActiveChar()
+				hasC1 := c.Base.Cons >= 1
+				// C1 ignores HP limit
+				hasPlayer := c.Core.Combat.Player().IsWithinArea(burstArea) && (char.HPCurrent/char.MaxHP() <= 0.7 || hasC1)
```

Comment:
nit: rename to something that communicates what it is doing better, like `needHeal`